docs/source/optim.rst

```diff
@@ -39,16 +50,23 @@ as optimization options for this group.
     between parameter groups.
 
 
-For example, this is very useful when one wants to specify per-layer learning rates::
+Consider the following example related to the distinct penalization of parameters.
+Remember that :func:`~torch.nn.Module.parameters` returns an iterable that 
+contains all learnable parameters, including biases and other 
+parameters that may need distinct penalization. To address this, one can specify
+individual penalization weights for each parameter group::
+
+    bias_params = [p for name,p in self.named_parameters() if 'bias' in name]
+    others = [p for name,p in self.named_parameters() if 'bias' not in name]
 
     optim.SGD([
-                    {'params': model.base.parameters()},
-                    {'params': model.classifier.parameters(), 'lr': 1e-3}
-                ], lr=1e-2, momentum=0.9)
+                    {'params': others },
+                    {'params': bias_params, 'weight_decay': 0}
+                ], weight_decay=1e-3, lr=1e-2)
```

Comment:
1e-3 is a really small weight_decay--is this intentional?

Reply:
No it's completely random. Can be changed if you believe it's too small

Reply:
Actually, just to be sure i'll change it to `1e-2`.

Reply:
people normally use 0.1 i believe btw